samples/ChangeFeedMigrationTool/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.DocumentDB" version="1.14.1" targetFramework="net47" />
-  <package id="Microsoft.Azure.DocumentDB.ChangeFeedProcessor" version="1.0.0" targetFramework="net47" />
+  <package id="Microsoft.Azure.DocumentDB" version="1.17.0" targetFramework="net47" />
```

Comment:
I'd stick to net45/net452 for the targetFramework as that's what the package is built for. 
@ealsur : Is there a reason why this has targetFramework=net47?

Reply:
@rnagpal your decision? merge or cancel?